lib/expressView.js

```diff
@@ -20,7 +20,30 @@ var format = require('util').format;
 var inherit = require('util').inherits;
 var debug = require('debug')(require('../package').name);
 
-var View = util.safeRequire('express/lib/view');
+var ExpressView = util.safeRequire('express/lib/view');
+
+function View(name, options) {
+  options = options || {};
+  this.name = name;
+  this.root = options.root;
+  var engines = options.engines;
+  this.defaultEngine = options.defaultEngine;
+  console.log(this.defaultEngine);
+
+  //var ext = this.ext = extname(name);
+  var ext = this.ext = '.' + this.defaultEngine;
```

Comment:
This not base the `ext` on the `name` and set the `ext` to the `defaultEngine` which in out case already has the variable `js` or `jsx`.